### PR TITLE
HAWKULAR-516 : Wizard for installing JDBC Driver

### PR DESCRIPTION
**Pull request by ammendonca**

### CODE REVIEW

File: console/src/main/scripts/bower.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "ngInfiniteScroll": "1.2.1",
     "bootstrap-select": "1.6",
     "hawkular-charts": "0.5.6",
-    "hawkular-ui-services": "0.8.2",
+    "hawkular-ui-services": "0.8.3",
```
Comment (mtho11) [inline console/src/main/scripts/bower.json]: Services has been busy this morning, the version you want is actually 0.8.5 now.

**Comment by mtho11** [comment]
Merged #476

**Comment by ppalaga** [comment]
@mtho11 this was not 100% ready to merge

**Comment by mtho11** [comment]
There were no comments stating it wasn't ready to merge, nor a label 'Dont merge Yet' so it gets merged ;)

**Comment by ppalaga** [comment]
Alex asked for review on IRC. Looks like it is safer to communicate here.

**Pull request merged**